modules/javafx.graphics/src/test/java/test/com/sun/javafx/pgstub/StubTextLayout.java

```diff
@@ -37,12 +37,17 @@
 
     @Override
     public boolean setContent(TextSpan[] spans) {
+        this.spans = spans;
+        this.text = null; /* Initialized in getText() */
+        this.nullFontSize = 10; // need a non-zero size
         return true;
     }
 
+    private TextSpan[] spans;
     private String text;
     private Font font;
     private int tabSize = DEFAULT_TAB_SIZE;
+    private int nullFontSize = 10;
```

Comment:
Setting `nullFontSize` to 10 by default has broken one of the existing unit tests (a rather fragile test it seems). I now see the following failure:

```
test.javafx.scene.chart.StackedBarChartTest > testSeriesAdd FAILED
    org.junit.ComparisonFailure: expected:<10 4[78 234 37 254 432 234 83 499 375 234 140] > but was:<10 4[60 218 35 238 415 218 80 465 360 218 135] >
        at org.junit.Assert.assertEquals(Assert.java:123)
        at org.junit.Assert.assertEquals(Assert.java:145)
        at test.javafx.scene.chart.StackedBarChartTest.testSeriesAdd(StackedBarChartTest.java:110)
```

If you change this to 0, which restores the previous default, leaving in place the setting to 10 in ` setContent(TextSpan[])`, then all tests pass.